systemtests/system.go

```diff
@@ -103,7 +99,7 @@ func NewSystemUnderTest(execBinary string, verbose bool, nodesCount int, blockTi
 		out:               os.Stdout,
 		verbose:           verbose,
 		minGasPrice:       fmt.Sprintf("0.000001%s", sdk.DefaultBondDenom),
-		projectName:       nameTokens[0],
+		projectName:       "simd",
```

Comment:
This change doesn't seem right

Reply:
ah right - this was an oddity i ran into when doing `testnet init-files` with my simd binary and simdv50 binary -- it would set my projectName to simdv50, even though simdv50's `init-files` was producing file paths ./testnet/node0/simd.

ill find another way to remedy this issue.